mkt/reviewers/views.py

```diff
@@ -112,7 +116,30 @@ def wrapper(request, *args, **kw):
         return decorator
 
 
-@reviewer_required
+def moderator_required(region=None):
+    """Requires the user to be logged in as a review moderator or admin.
+    A moderator is someone who is in one of the groups with the following
+    permissions:
+
+        Apps:ModerateReview
+
+    """
+    def decorator(f):
+        @login_required
+        @functools.wraps(f)
+        def wrapper(request, *args, **kw):
+            if (acl.action_allowed(request, 'Apps', 'ModerateReview')):
+                return f(request, *args, **kw)
+            else:
+                raise PermissionDenied
+        return wrapper
+    # If decorator has no args, and is "paren-less", it's callable.
+    if callable(region):
+        return decorator(region)
+    else:
+        return decorator
```

Comment:
I seem to remember a `permission_required` decorator that could be used instead of making another one?